gatsby-config.js

```diff
@@ -4,9 +4,9 @@ require('dotenv').config({ path: `.env/env.${activeEnv}` });
 
 module.exports = {
   siteMetadata: {
-    title: 'Ovio | Connecting Tech Minds and Impact Makers',
+    title: 'Ovio | Empowering developers to become active open source contributors',
     // eslint-disable-next-line max-len
-    description: 'Ovio revolutionizes skill-based volunteering for the tech industry by bringing together social impact organizations, passionate people, and companies to make tangible impact easy and rewarding for all.',
+    description: 'We are building a more accessible and inclusive open-source ecosystem, empowering millions of developers to become active contributors.',
```

Comment:
@tiphaineboyer 

This modification is mostly for shared info. Is it what you want to change?

![image](https://user-images.githubusercontent.com/40799514/76646255-f16aed00-655a-11ea-9519-2567a42934ce.png)